var/spack/repos/builtin/packages/curl/package.py

```diff
@@ -239,3 +260,54 @@ def with_or_without_secure_transport(self, activated):
                 return "--with-darwinssl"
             else:
                 return "--without-darwinssl"
+
+
+class NMakeBuilder(NMakeBuilder):
```

Comment:
I assume this PR depends on https://github.com/spack/spack/pull/34659

Reply:
Yes. I forgot to specify that when I switched this PR over to the new builders as this PR has existed for quite some time independently of the multi build system feature/those changes.